hw/ip/otbn/otbn.core

```diff
@@ -42,24 +36,14 @@ filesets:
       - lowrisc:prim:assert
       - lowrisc:prim:util
       - lowrisc:prim:ram_1p_adv
+      - lowrisc:ip:otbn_pkg
+      - lowrisc:dv:otbn_model
```

Comment:
does IP need the DV model?

Reply:
Sort of: the top-level `otbn.sv` code has a "use a model instead of the RTL" mode, controlled with a define and a parameter. I think Philipp is using this to help with integration work. Eventually, that will probably go away.

Note: This PR doesn't change that at all; it just splits up the core files, so the dependency becomes more explicit.

Reply:
sounds good.